TerminalDocs/customize-settings/color-schemes.md

```diff
@@ -3,16 +3,44 @@ title: Windows Terminal Color Schemes
 description: Learn how to create color schemes for Windows Terminal.
 author: cinnamon-msft
 ms.author: cinnamon
-ms.date: 05/19/2020
+ms.date: 7/28/2020
 ms.topic: how-to
 ms.service: terminal
 ms.localizationpriority: high
 ---
 
 # Color schemes in Windows Terminal
 
+Windows Terminal lets you define your own color schemes, either by using the built-in preset themes, or by creating your own theme from scratch. To change  themes, you'll need to edit the settings.json file in an editor such as [Visual Studio Code](https://code.visualstudio.com/).
```

Comment:
```suggestion
Windows Terminal lets you define your own color schemes, either by using the built-in preset schemes, or by creating your own scheme from scratch. To change schemes, you'll need to edit the settings.json file in an editor such as [Visual Studio Code](https://code.visualstudio.com/).
```

Reply:
For context, we'll be adding themes later, so this will help avoid confusion in the future. 😊